CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ All notable changes to this project will be documented in this file based on the
 - In QueryString is not allowed to use fields parameters in conjunction with default_field parameter. This is not well documented, it's possibile to understand from [Elasticsearch tests :  QueryStringQueryBuilderTests.java](https://github.com/elastic/elasticsearch/blob/6.0/core/src/test/java/org/elasticsearch/index/query/QueryStringQueryBuilderTests.java#L917) [#1352](https://github.com/ruflin/Elastica/pull/1352)   
 - Index mapping field of type [*'string'*](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/string.html) has been removed from Elasticsearch 6.0 codebase [#1353](https://github.com/ruflin/Elastica/pull/1353)
 - The [created and found](https://github.com/elastic/elasticsearch/pull/25516) fields in index and delete responses became obsolete after the introduction of the result field in index, update and delete responses [#1354](https://github.com/ruflin/Elastica/pull/1354) 
- 
+- Disallow [include_in_all](https://github.com/elastic/elasticsearch/pull/22970) for 6.0+ indices [#1362](https://github.com/ruflin/Elastica/pull/1362)
```

Comment:
There is no change for this in Elastica as far as I can see.

Reply:
right. I removed the entry in the changelog.